src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/tests/test_cloudformation.py

```diff
@@ -6,10 +6,11 @@
 import os
 import boto3
 from pytest import fixture
+from unittest.mock import patch
```

Comment:
Is this import needed?

Reply:
Good catch! Thanks for that, I will remove it.